client/my-sites/plugins/main.jsx

```diff
@@ -164,11 +166,17 @@ export default React.createClass( {
 		}
 	},
 
-	getSelected() {
+	getSelected( isWpCom ) {
 		if ( ! this.state.plugins ) {
 			return [];
 		}
-		return this.state.plugins.filter( this.filterSelection.selected );
+
+		return this.state.plugins.filter( plugin => {
+			if ( plugin.selected ) {
+				return isWpCom ? plugin.wpcom : ! plugin.wpcom
+			}
+			return false;
```

Comment:
What's this supposed to return?

I assume that `plugin.wpcom` is a boolean value but I'm not sure.

It looks like this:

```js
const { selected, wpcom } = plugin;

selected && wpcom.negateIf( isWpCom )
```